dune

```diff
@@ -1 +1 @@
-(ignored_subdirs (bootstrap))
+(dirs :standard \ bootstrap)
```

Comment:
I assumed these two would behave equivalently, but perhaps this should in fact be `(data_only_dirs bootstrap)`

Reply:
let's try ^^

Reply:
it didn't work :(

Reply:
Your latest commit looks like it should work, thanks - I shouldn't have ever switched that to `vendored_dirs`... the point is that it's supposed to ignore any of the pkg build directories (because they'll be dirty) and it should find the packages in `bootstrap/` because that's the system root.